### DISCUSSION TITLE
does Gramps.js satisfies some of the old design docs on Gramps-Project.org?

### DISCUSSION THREAD

**Discussion by emyoulation** [gramps-project/gramps-web#321] [Q&A] [answered]
The [Gramps Server Mode](https://www.gramps-project.org/wiki/index.php/Gramps_Server_Mode) wiki article was spun out of [GEPS 013: Gramps Webapp](https://www.gramps-project.org/wiki/index.php/GEPS_013:_Gramps_Webapp).

It seems like the recent development should be an answer to the questions brought up in both articles.  Could someone review those 2 articles and see if they should be updated with references to the new development?

**Comment by DavidMStraub** [comment]
Hi,

I suggest to put an info box on top of both wiki articles and say: "This page is part of an obsolete discussion of a Gramps web app and has been superseded by Gramps Web" (and link to https://gramps-project.org/wiki/index.php/Addon:Gramps_Web).

The motivation in GEPS 013 is essentially the same motivation as Gramps Web, but the architecture is very different. One important difference is that the server code and the user interface code are separated, while GEPS 013 would have implemented both in Django.

The "Gramps Server Mode" is also obsolete - it is replaced by the Gramps Web API (https://github.com/gramps-project/gramps-webapi/).

**Comment by emyoulation** [comment, marked as answer]
Is this appropriate?  (Although "branch/gep/geps-013-server SVN" was apparently deleted from GitHub.)

[GEPS 13](https://www.gramps-project.org/wiki/index.php/GEPS_013:_Gramps_Webapp)
**WebApp has been superseded**
This GEPS page is part of an obsolete discussion of a Gramps web app and has been superseded by [Gramps Web](https://www.gramps-project.org/wiki/index.php/Addon:Gramps_Web), a server-based variant of Gramps was released in late 2022.
[Gramps Server Mode](https://www.gramps-project.org/wiki/index.php/Gramps_Server_Mode) is also obsolete. It is replaced by the [Gramps Web API](https://github.com/gramps-project/gramps-webapi/) in March 2023.
The architecture is very different than specified in this GEPS. One important difference is that the server code and the user interface code are separated, rather than both implemented in Django.

[Gramps Server Mode](https://www.gramps-project.org/wiki/index.php/Gramps_Server_Mode)
**Gramps Server Mode has been superseded**
Gramps Server Mode is obsolete and was replaced by the [Gramps Web API](https://github.com/gramps-project/gramps-webapi/) in March 2023.
This page is part of an obsolete discussion of [GEPS013:Gramps WebApp](https://www.gramps-project.org/wiki/index.php/GEPS_013:_Gramps_Webapp) which has been superseded by [Gramps Web](https://www.gramps-project.org/wiki/index.php/Addon:Gramps_Web), a server-based variant of Gramps released in late 2022.
While this has been abandoned/superseded, these notes are left here in case they might be useful to someone. The prototype code for this functionality is in the branch/gep/geps-013-server SVN.

**Reply by DavidMStraub** [reply]
Perfect IMO.
